Classes/EventListener/EventListenerLocator.php

```diff
@@ -39,7 +39,7 @@ class EventListenerLocator
     protected $eventTypeService;
 
     /**
-     * @var array
+     * @var array in the format ['<eventType>' => ['<listenerClassName>' => '<listenerMethodName>', '<listenerClassName2>' => '<listenerMethodName2>', ...]]
```

Comment:
I tweaked the internal format in order to make it easier to determine a single listener from a given classname and/or the event types that are handled by the listeners of a specific class